Add example documentation on reading/writing a frame file (#1552)
gwpy/gwpy: docs/timeseries/io.rst

**If the output file already exists it will be overwritten.**

.. note::

>    When writing a timeseries to a `gwf` file, the `TimeSeries.name` property is used to determine the `channel`. Therefore, if you want to write a file and then read it back in, you should first set the name to the chosen channel, e.g.

Reviewer: Minor rewrite of text to give some more detail:

```suggestion
   When writing a timeseries to a GWF, the :attr:`TimeSeries.name` property is used for the `name` variable of
   the GWF data structures (`FrProcData` and `FrVect`).
   So, if you want to write a file and then read it back in, you must ensure that the :attr:`~TimeSeries.name` property
   is correctly assigned, e.g:
```

   .. code-block:: python

> 

Reviewer: Can this code block _define_ a `TimeSeries`, so that the reader understands a circumstance where the `name` property is unassigned?

      >>> channel = "L1:CHANNEL_NAME"
      >>> output_file = "output.gwf"

>       >>> data.channel = channel

Reviewer: Set the `name` attribute:

```suggestion
      >>> data.name = channel
```

      >>> data.write(output_file)
      >>> data = TimeSeries.read(output_file, channel)

> 

Reviewer: ```suggestion
```
